essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-use-case-software-engineer.md

```diff
@@ -5,7 +5,7 @@
 ## Introduction
 Welcome, fellow software engineers! Today, I want to share with you how I use Appflowy to stay organized and productive in my work. As software engineers, we all know how important it is to keep track of our projects, deadlines, and tasks. That's where Appflowy comes in! With its simple and intuitive design, Appflowy makes it easy to stay on top of everything.
 
-![Getting started page](https://user-images.githubusercontent.com/89600478/235852123-0105d13e-b05e-4df0-b136-4825ca817351.png)
+![Getting started page](https://github.com/AbubakrChan/AppFlowy-Docs/assets/89600478/d5776555-f530-4aad-a1e8-640ebebd24c9)
```

Comment:
You can move the assets to `.gitbook\assets`, and then use:
```suggestion
![Getting started page](https://github.com/AppFlowy-IO/AppFlowy-Docs/tree/main/.gitbook/assets/89600478/d5776555-f530-4aad-a1e8-640ebebd24c9)
```I would assume this to work.

Reply:
Thank you for bringing this to my attention!